test/forge/ReallocateWithdrawTest.sol

```diff
@@ -59,7 +59,7 @@ contract ReallocateWithdrawTest is IntegrationTest {
         assertEq(_idle(), INITIAL_DEPOSIT, "idle");
     }
 
-    function testReallocateWithdrawInconsistentAsset() public {
+    function testReallocateWithdrawMarketNotEnabled() public {
         ERC20Mock loanToken2 = new ERC20Mock("loan2", "B2");
         allMarkets[0].loanToken = address(loanToken2);
```

Comment:
might need to be updated as well and/or fuzz

Reply:
Why would you want to update it?
And fuzzing it is not an option, so we simply deploy a second token (or show me the way!)

Reply:
Markets with an inconsistent asset is a subset of markets that are not enabled, so i'd propose to test the function on a larger set like fuzzing the market params. But this is not really important

Reply:
Ah but you're right there's no call involved so we could fuzz

Reply:
But we need a real loan token to supply on behalf of the vault